build/build.sh

```diff
@@ -43,3 +44,20 @@ wkhtmltopdf \
   --quiet \
   output/index.html \
   output/manuscript.pdf
+
+echo "Exporting Word Docx manuscript"
+pandoc --verbose \
+--smart \
+--from=markdown \
+--to=docx \
+--filter pandoc-fignos \
+--filter pandoc-tablenos \
+--bibliography=$BIBLIOGRAPHY_PATH \
+--csl=$CSL_PATH \
+--reference-docx=$DOCX_PATH \
```

Comment:
> Can you explain what this is? What happens if you don't pass it?
not passing `--reference-docx=$DOCX_PATH` makes some default formatting options that the user would change. I think it uses the Normal template in Word. Thus font in Calabri font instead of Times New Roman. Some of the Headers are Blue instead of Black. Its not necessary, but users can change the style settings and the next time they generate the word doc it will use those styles. Allowing users to change the styles necessary for different journals or collaborators.